components/service/sync-autofill/src/main/java/mozilla/components/service/sync/autofill/AutofillCreditCardsAddressesStorage.kt

```diff
@@ -94,6 +95,25 @@ class AutofillCreditCardsAddressesStorage(
     }
 }
 
+/**
+ * An implementation of [CreditCardsAddressesStorage] back by the application-services' `autofill`
+ * library.
+ */
+class AutofillCreditCardsAddressesCrypto() : CreditCardsAddressesCrypto {
```

Comment:
I think you copied the comment here. :)

I think it might be nice to keep files small and move this class into its own file along with the tests.